studies/MultiObjectiveOptimizer.jl

```diff
@@ -27,6 +27,7 @@ Base.@kwdef mutable struct FUSEparameters__ParametersStudyMultiObjectiveOptimize
     n_workers::Entry{Int} = study_common_parameters(; n_workers=missing)
     file_save_mode::Switch{Symbol} = study_common_parameters(; file_save_mode=:safe_write)
     release_workers_after_run::Entry{Bool} = study_common_parameters(; release_workers_after_run=true)
+    run_in_safe_mode :: Entry{Bool} = Entry{Bool}("-", "Runs the optimization in a loop restarting the workers every 5 generations"; default=false)
```

Comment:
`run_in_safe_mode` is not a good name. Maybe `restart_workers_after_n_generations` ? also, this should be an `Int`, not a `Bool` with the ability to set after how many generations the workers would be restarted. Default should be `0` to mean no restart.